vmdb/app/models/miq_group.rb

```diff
@@ -64,9 +64,9 @@ def self.allows?(group, options={})
   def self.add(attrs)
     group = self.new(attrs)
     if group.resource.nil?
-      groups = self.find(:all, :conditions => {:resource_id => nil, :resource_type => nil}, :order => "sequence DESC")
+      groups = self.where(:resource_id => nil, :resource_type => nil).order("sequence DESC")
     else
-      groups = group.resource.miq_groups.find(:all, :order => "sequence DESC")
+      groups = group.resource.miq_groups.order("sequence DESC")
     end
```

Comment:
You don't even need all the groups, but this works just fine...thinking it could be cleaned up as:

```ruby
    groups =
      if group.resource.nil?
        where(:resource_id => nil, :resource_type => nil)
      else
        group.resource.miq_groups
      end
    last_group = groups.order("sequence DESC").first
    group.sequence = last_group.nil? ? 1 : last_group.sequence + 1
```

Also, seeing a possible race condition here... cc @gmcculloug, @bzwei